packages/create-pantheon-decoupled-kit/src/templates/next-wp-search-api-addon/pages/search/[[...searchAlias]].jsx

```diff
@@ -1,26 +1,95 @@
 import { NextSeo } from 'next-seo';
```

Comment:
There is probably an argument that could be made to make this a shared template with the search page in the `next-drupal` starter. I chose not to because I feel like the logic in `getServerSideProps` as well as how the data shapes are destructured both differ enough from one another to warrant their own files. They do still share some bits though. The `next-drupal` file is [here](https://github.com/pantheon-systems/decoupled-kit-js/blob/feat/wp-search-addon/packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/pages/search/%5B%5B...searchAlias%5D%5D.jsx) if you want to compare the two and weigh in any thoughts.

Reply:
> I chose not to because I feel like the logic in getServerSideProps as well as how the data shapes are destructured both differ enough from one another to warrant their own files. 

Sounds reasonable to me.